plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -268,7 +268,15 @@ public void setImageName(final String imageName) {
 		final DockerImageURI uri = new DockerImageURI(imageName);
 		setResourceName(dockerImage2OpenshiftResourceConverter.convert(uri));
 	}
-	
+
+	@Override
+	public void setImageName(String imageName, boolean forceUpdate) {
+		if(forceUpdate && this.imageName != null && this.imageName.equals(imageName)) {
```

Comment:
usually there is a simpler fix for this: simply use null as old value when firing.